README.md

```diff
@@ -98,6 +102,41 @@ export default class Ember.Component.extend({
 }) { }
 ```
 
+### ES6 Compatible
+
+`ember-strong-attr` exposes one function to declare strong attributes on
+`Ember.Component`
+
+- `declareStrongAttrs(attrs, component)`, it returns the modified `component` that was passed in.
+
+Example:
+
+```js
+import Ember from 'ember';
+import { declareStrongAttrs } from 'ember-strong-attrs';
+import Person from '../models/person';
+
+export default declareStrongAttrs({
+  'myRequiredAttr': {
+    type: String,
+    required: true
+  },
+  'myOptionalAttr': {
+    type: String,
+    required: false
+  },
+  'myOtherOptionalAttr': {
+    type: String
+  }
+  'myPersonAttr': {
+    type: Person,
+    required: true,
+  }
+}, Ember.Component.extend({
+  // ... your methods and props
+}));
+```
```

Comment:
That is actually really nice! I was trying to achieve what you just suggested but couldn't see the final step.

The end goal of this API was to allow people on ES6 to use ember-strong-attrs and then easily transition to ES7 decorators once that spec is formally supported. I think the DSL you listed would make the transition super trivial!

Expect to see an update tonight!